src/views/Sjofn/Practice41.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div>
+    Practice 4-1
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Practice2',
+  data () {
+    return {
+      number: '',
+      one: '',
+      two: '',
+      three: '',
+      four: ''
```

Comment:
You don't need any of these data attributes, because all of methods (or 'functions', if you wish) don't work with any values from component's scope. Each method either receives some value or creates it inside.
Please, remove the data-attributes and check that everything still works as expected.

P.S.:
please, also check this article: https://learn.javascript.ru/function-basics